archive4pl.c

```diff
@@ -227,15 +311,15 @@ static PL_blob_t archive_blob =
 { PL_BLOB_MAGIC,
   0,
   "archive",
-  release_archive,
-  compare_archives,
-  write_archive,
-  acquire_archive
+  ar_w_release_cb,
+  ar_w_compare_cb,
+  ar_w_write_cb,
+  ar_w_acquire_cb
 };
 
 
 static int
-get_archive(term_t t, archive_wrapper **arp)
+term_to_archive_wrapper(term_t t, archive_wrapper **arp)
```

Comment:
I'm not that happy with this renaming.   The convention to use `get_<type>(term_t, <type> **obj)`  is used everywhere in the core and packages.

Reply:
I wasn't aware of this convention. I'll revert this change.